modules/aurora-postgres/ssm.tf

```
@@ -1,5 +1,5 @@
 locals {
-  ssm_path_prefix = format("/%s/%s", var.ssm_path_prefix, module.cluster.id)
+  ssm_path_prefix = var.ssm_cluster_name_override ? format("/%s", var.ssm_path_prefix) : format("/%s/%s", var.ssm_path_prefix, module.cluster.id)
```

```suggestion
  ssm_path_prefix = var.ssm_cluster_name_override ? format("/%s/%s", var.ssm_path_prefix, var.ssm_cluster_name_override) : format("/%s/%s", var.ssm_path_prefix, module.cluster.id)
```

I might be missing something from this PR but wouldn't it make sense to utilize the override being provided?

---

make sense, updated